client/deck-validator.js

```diff
@@ -281,9 +285,32 @@ class DeckValidator {
             requiredPlots: 7,
             maxDoubledPlots: 1
         };
+
         let factionRules = this.getFactionRules(deck.faction.value.toLowerCase());
         let agendaRules = this.getAgendaRules(deck);
-        return this.combineValidationRules([standardRules, factionRules].concat(agendaRules));
+        let rookeryRules = this.getRookeryRules(deck);
+
+        return this.combineValidationRules([standardRules, factionRules, rookeryRules].concat(agendaRules));
+    }
+
+    getRookeryRules() {
+        return {
+            mayInclude: () => true,
```

Comment:
This shouldn't be here. It's explicitly allowing every single card in the full deck, regardless of other `mayInclude`s.

It also might be nice to add a rule below that the player doesn't include agendas in the rookery.